apps/remote_control/test/lexical/remote_control/completion_test.exs

```diff
@@ -0,0 +1,47 @@
+defmodule Lexical.RemoteControl.CompletionTest do
+  alias Lexical.Document
+  alias Lexical.Document.Position
+  alias Lexical.RemoteControl.Completion
+
+  import Lexical.Test.CursorSupport
+  import Lexical.Test.CodeSigil
+
+  use ExUnit.Case, async: true
+
+  describe "struct_fields/2" do
+    test "it should move the cursor into struct arguments and return the correct fields" do
```

Comment:
I disagree with this; we should most definitely _not_ "move the cursor" anywhere; from where the cursor is positioned here, we should be completing struct modules, not fields.

Reply:
Moving the cursor is the previous method, I forgot to update the description.